src/Chapter13/Listing13.16.PassingADelegateWithAnExpressionLambda.cs

```diff
@@ -1,9 +1,9 @@
-﻿// TODO: Update listing in Manuscript
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Listing13_16
+﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Listing13_16
 {
     using System;
     using Listing13_11;
-
+    #region INCLUDE
+    #region EXCLUDE
```

Comment:
Rather an include with two excludes to strip of the beginning and end, wouldn't it be simpler to just move the #region INCLUDE to just surround the inner section?